script/tool/lib/src/custom_test_command.dart

```diff
@@ -43,10 +43,19 @@ class CustomTestCommand extends PackageLoopingCommand {
 
     // Run the custom Dart script if presest.
     if (script.existsSync()) {
-      final int exitCode = await processRunner.runAndStream(
+      // Ensure that dependencies are available.
+      final int pubGetExitCode = await processRunner.runAndStream(
```

Comment:
`runAndStream` prints to stdout and stderr?  If so this looks good.  If not we should make sure we output that information when the exit code != 0.

Reply:
Yep, that's what the "AndStream" is supposed to capturing:

> /// Run the [executable] with [args] and stream output to stderr and stdout.